### ISSUE TITLE
Crossfield training fundamental bug in FrameField

### ISSUE THREAD

**Issue by patriksabol** [dsgoficial/pytorch_segmentation_models_trainer#173]
Hi, 
thanks for great framework! However there is huge fundamental bug in FrameField approach. 
First [here](https://github.com/dsgoficial/pytorch_segmentation_models_trainer/blob/7e4839273c4744b47ca47c4c8c6c44c736c6dfca/pytorch_segmentation_models_trainer/dataset_loader/dataset.py#L417)
crossfield is treated as mask during loading, thus, in [here](https://github.com/dsgoficial/pytorch_segmentation_models_trainer/blob/7e4839273c4744b47ca47c4c8c6c44c736c6dfca/pytorch_segmentation_models_trainer/dataset_loader/dataset.py#L155) all anglefield are vanished and set to value one.
Moreover, any affine augmentation from albumentation is not usable, because anglefield has to be treated in different way for affine transformations, as in the original FrameField repo f.e. [rotate_anglefield](https://github.com/Lydorn/pytorch_lydorn/blob/f86bc9cfaa009ea73bd6c07d8f4b2d07c8f65431/torch_lydorn/torchvision/transforms/functional.py#L167)

**Comment by phborba** [comment]
Thank you so much for your report! You are right, I'll fix these bugs !

**Comment by patriksabol** [comment]
Hi,
I have also found very small but critical bug. I was not able to train properly crossfield itself. And the problem was in [this](https://github.com/dsgoficial/pytorch_segmentation_models_trainer/blob/7e4839273c4744b47ca47c4c8c6c44c736c6dfca/pytorch_segmentation_models_trainer/utils/polygon_utils.py#L116) line for crossfield generation.
In your code is this line 
` edge_angle_array = np.angle(edge_vect_array[:, 0] + 1j * edge_vect_array[:, 1])`
but comparing to the origin codebase [here](https://github.com/Lydorn/pytorch_lydorn/blob/f86bc9cfaa009ea73bd6c07d8f4b2d07c8f65431/torch_lydorn/torchvision/transforms/angle_field_init.py#L54), the indices are switched as follows:
`edge_angle_array = np.angle(edge_vect_array[:, 1] + 1j * edge_vect_array[:, 0])  # ij coord sys`
it took me week to find this bug :smile: 
This bug leads to false encoding anglefield in dataset

**Comment by phborba** [comment]
Thank you so much for your help. I've fixed the frame field loading issue and the creation issue. Later on I'll treat the problems with rotation and publish a new release for the library.
